doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1022 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        preds_sim = model(torch.Tensor(sim_embedding).to(device)).detach().cpu().numpy().argmax(axis=1)
</pre><p>Consider calling the model only on top k instances and not the entire dataset. Same applies for ImageNet and MNIST. Check their comments for more details and quick fix.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
hmm ... right ... then consider doing it in batches.

Reply:
If you call the model on all the instances, please consider doing it in batches. Can't pass 1000 examples through the model on a regular GPU

Reply:
<p>Actually, I wasn't getting the predictions twice, so it is actually necessary to call the model to produce the plots with the scores averaged for each predicted class. After discussing with Robert offline I left it as it was. </p>